src/routing.ts

```diff
@@ -0,0 +1,47 @@
+export type RouteCategories = 'Statystyki' | 'Menu' | 'Magazyn' | 'Dostęp' | 'Ustawienia';
```

Comment:
You made very nice types in this file, I like it. 👍

Reply:
Thanks bro, I learn from the best! :)